packages/main/src/Avatar.ts

```diff
@@ -281,6 +295,7 @@ class Avatar extends UI5Element implements ITabbable {
 	_onclick?: (e: MouseEvent) => void;
 	static i18nBundle: I18nBundle;
 	_handleResizeBound: ResizeObserverCallback;
+	_interactive!: boolean;
```

Comment:
If we don't initialize the class member inside construct it have undefined as default value so defining it with `!` is wrong because it excludes the undefined as possible value. If you add this._interactive inside constructor that exclamation mark will be redundant